### ISSUE TITLE
About the code in DGL

### ISSUE THREAD

**Issue by MrZealand** [Jhy1993/HAN#33]
The code in [https://github.com/dmlc/dgl/blob/master/examples/pytorch/han/model.py](url) Line 19,Line21.
`beta = torch.softmax(w, dim=1)`
`return (beta * z).sum(1)`

The dim should be 0 ?

**Comment by mongoose54** [comment]
The features are on `dim=1`  . So it is correct.

**Comment by lipingcoding** [comment]
I found code in dgl obtains inferior performance. Do you have any idea? Please ref this issue! Thanks if you have time

**Comment by yzh119** [comment]
@lipingcoding what doo you mean by performance? speed or final accuracy?
